content/800-data-platform/200-pulse/200-getting-started.mdx

```diff
@@ -20,16 +20,98 @@ To participate in Pulse's Early Access program, you need to meet the following p
 
 You will also need a database with the following configurations:
 
-- Postgres version 12+.
+- PostgreSQL version 12+.
 - Ensure your database is publicly accessible.
-- Enable [logical replication](https://www.postgresql.org/docs/current/logical-replication-quick-setup.html) on your database.
+- [Set the `wal_level` setting in PostgreSQL to `logical`](/data-platform/pulse/getting-started#wal_level)
 - A database superuser that can be used for connections inside Pulse.
 
-## 1. Enable Pulse in a Cloud Project
+## 1. Database setup
+
+Depending on your database provider, the setup may vary. This section will attempt to cover the most common providers. If you have a provider that isn't covered, please reach out to a developer advocate at Prisma for assistance.
+
+### General database configuration
+
+#### <inlinecode>[wal_level](https://www.postgresql.org/docs/current/runtime-config-wal.html)</inlinecode>
+
+Some providers may not allow direct access to this setting. If you are unable to change this setting, please refer to the provider-specific guides for further assistance.
+
+```SQL
+  ALTER SYSTEM SET wal_level = logical;
```

Comment:
btw, any idea why the syntax highlighting doesn't work?  is it because of the leading spaces?

<img width="418" alt="image" src="https://github.com/prisma/docs/assets/4058327/17e4d9c8-c491-4c10-923b-ea43644e974b">

Reply:
![image](https://github.com/prisma/docs/assets/36578389/99bdc11e-4217-46ba-b1cd-8ec99a9e28cb)

Doesn't seem like it. I removed the leading space from every SQL statement, which still doesn't highlight.  Also doesn't do anything when set to PostgreSQL

Reply:
not sure if that's the issue but maybe try lowercasing `sql`? this works on other pages in the docs, e.g. here: https://www.prisma.io/docs/guides/other/advanced-database-tasks/data-validation/postgresql#2-adding-a-table-with-a-single-check-constraint-on-a-single-column so this should generally be possible.